nixos/modules/services/misc/open-webui.nix

```diff
@@ -12,55 +12,70 @@ in
 {
   options = {
     services.open-webui = {
-      enable = lib.mkEnableOption "Enable open-webui, an interactive chat web app";
+      enable = lib.mkEnableOption "Open-WebUI server";
       package = lib.mkPackageOption pkgs "open-webui" { };
 
       stateDir = lib.mkOption {
         type = types.path;
         default = "/var/lib/open-webui";
-        description = "State directory of open-webui.";
+        example = "/home/foo";
+        description = "State directory of Open-WebUI.";
       };
 
       host = lib.mkOption {
         type = types.str;
-        default = "localhost";
-        description = "Host of open-webui";
+        default = "127.0.0.1";
+        example = "0.0.0.0";
+        description = ''
+          The host address which the Open-WebUI server HTTP interface listens to.
+        '';
       };
 
       port = lib.mkOption {
         type = types.port;
```

Comment:
Is `types.port` a real type? I had to use `types.ints.u16` in ollama's module since I couldn't find a port type.

Reply:
Yes, it's used in many places in `nixos/modules/services` :)

Reply:
Do you think it's worth switching `types.ints.u16` to `types.port` in ollama's module?

Reply:
Yes, just did it in this PR!